modules/serverless-send-events-kn.adoc

```diff
@@ -0,0 +1,43 @@
+[id="serverless-send-events-kn_{context}"]
+= Sending events
+
+You can use the `kn event send` command to send an event. The events can be sent either to publicly available addresses or to addressable resources inside a cluster, such as Kubernetes services, as well as Knative services, brokers, channels, and so on. The command uses the same builder-like interface as the `kn event build` command.
+
+.Procedure
+
+* Send an event:
++
+[source,terminal]
+----
+$ kn event send --field <field-name>=<value> --type <type-name> --id <id> --to-url <url> --to <cluster-resource>
+----
+** The `--field` option adds data to the event as field-value pairs.
+** The `--type` option specifies the type of the event.
+** The `--id` option specifies the ID of the event.
+** You need to use either the `--to-url` option to specify the destination URL or the `--to` option to specify the destination Kubernetes resource.
++
```

Comment:
Also, using option `-n, --namespace` would be required in most use-cases, so it's worth to motion IMHO. By default, is taken from current kube context, similar to `kn` or `kubectl`.